src/dumpulator/dumpulator.py

```diff
@@ -1178,7 +1181,7 @@ def _hook_mem(uc: Uc, access, address, size, value, dp: Dumpulator):
                 assert value == dp.exception.memory_value
 
                 # Delete the code hook
-                uc.hook_del(int(dp.exception.code_hook_h))
+                uc.hook_del(dp.exception.code_hook_h.value)

```

Comment:
Shouldn't `exception.code_hook_h` just be an `int` as unicorn returns it, to not have to do these gymnastics?

Reply:
As I understood originally the variable was supposed to use `exception.code_hook_h` (from the TODO); although I agree that adding a comment and using the value itself could also work - although less verbose.

Reply:
Fixed by #[dcfccad](https://github.com/mrexodia/dumpulator/pull/47/commits/dcfccad8cb57b045c6fe5a3cecb60067c9920f89)